source/ui/ActivityItem/index.jsx

```diff
@@ -104,23 +158,50 @@ const ActivityItem = ({
             />
           )
       }
-
       <div className={classes.leftContainer}>
         <Typography variant="h5">
           {getTitle(type, currency, swapData, plug, t)}
         </Typography>
-        <Typography variant="subtitle2">
-          {getStatus(status, classes, t)}{getSubtitle(type, status, date, to, from, t)}
+        <Typography
+          variant="subtitle2"
+          onClick={handleClickCopy}
+          onMouseOver={() => setShowTooltip(true)}
+          onMouseLeave={() => setShowTooltip(false)}
+        >
+          {getStatus(status, classes, t)}{getDate(status, date)}
+          <Tooltip
+            classes={{ tooltipPlacementBottom: classes.tooltip }}
+            title={tooltipText}
+            arrow
+            open={showTooltip || copied}
+            placement="bottom"
+          >
+            <span>{getSubtitle(type, to, from, t)}</span>
+          </Tooltip>
         </Typography>
       </div>
       <div className={classes.rightContainer}>
-        <Typography variant="h5">
-          <NumberFormat value={showSwap ? swapData.amount : amount} displayType="text" thousandSeparator="," suffix={` ${showSwap ? swapData.currency.value : currency.value}`} decimalScale={5} />
-        </Typography>
-        <Typography variant="subtitle2">
-          <NumberFormat value={showSwap ? swapData.value : value} displayType="text" thousandSeparator="," prefix="$" suffix=" USD" decimalScale={2} />
-        </Typography>
+        <div>
+          <Typography variant="h5">
+            <NumberFormat value={showSwap ? swapData.amount : amount} displayType="text" thousandSeparator="," suffix={` ${showSwap ? swapData.currency.value : currency.value}`} decimalScale={5} />
```

Comment:
got an empty space in classname

Reply:
its intended to leave a space after the number